package.json

```
@@ -31,5 +31,9 @@
     "eslint-plugin-bpmn-io": "^0.10.0",
     "mocha": "^6.2.2",
     "npm-run-all": "^4.1.5"
+  },
+  "engines": {
+    "node": ">= 18.0.0",
+    "npm": ">= 8.0.0"
```

We still test this one on Node 10, so don't have to be as restrictive with the version numbers

```suggestion
    "node": ">= 10.0.0",
    "npm": ">= 6.0.0"
```